lib/prompt.js

```diff
@@ -114,7 +114,7 @@ class Prompt extends Events {
 
     let result = this.state.error || await this.validate(this.value, this.state);
     if (result !== true) {
-      let error = this.symbols.pointer + ' ' + result.trim();
+      let error = typeof result === 'string' ? this.symbols.pointer + ' ' + result.trim() : 'Your input is invalid.';
```

Comment:
Can you change this line to:

```js
let error = this.symbols.pointer + ' ' + (typeof result === 'string' ? result.trim() : 'Invalid input');
```

Reply:
Even better, put that line in a function named getError(result). Easier on the reader to digest what's happening.